resources/measures/advanced_rtu_control/measure.rb

```diff
@@ -0,0 +1,335 @@
+					  tot_oa_cfm = floor_area * tot_oa_cfm_per_ft2
+
+					  # if both per-area and per-person are present, does not need to be modified
+					  if !dsn_oa.outdoorAirFlowperPerson.zero? & !dsn_oa.outdoorAirFlowperFloorArea.zero?
+						next
+					  
+					  # if both are zero, skip space
+					  elsif dsn_oa.outdoorAirFlowperPerson.zero? & dsn_oa.outdoorAirFlowperFloorArea.zero?
+						runner.registerInfo("Space '#{space.name}' has 0 outdoor air per-person and per-area rates. DCV may be still be applied to this air loop, but it will not function on this space.")
+						next
+					  
+					  # if per-person or per-area values are zero, set to 10 cfm / person and allocate the rest to per-area
+					  elsif dsn_oa.outdoorAirFlowperPerson.zero? || dsn_oa.outdoorAirFlowperFloorArea.zero?
+						puts "========Before Per Person========="
+						puts "#{space.name}"
+						puts "people per m2", people_per_m2
+						puts "Per-person", dsn_oa.outdoorAirFlowperPerson * people_per_m2
+						puts "Per-area", dsn_oa.outdoorAirFlowperFloorArea
+						puts "Total OA", tot_oa_per_m2
+
+						if dsn_oa.outdoorAirFlowperPerson.zero?
+						  runner.registerInfo("Space '#{space.name}' per-person outdoor air rate is 0. Using a minimum of 10 cfm / person and assigning the remaining space outdoor air requirement to per-area.")
+						elsif dsn_oa.outdoorAirFlowperFloorArea.zero?
+						  runner.registerInfo("Space '#{space.name}' per-area outdoor air rate is 0. Using a minimum of 10 cfm / person and assigning the remaining space outdoor air requirement to per-area.")
+						end
+
+						# default ventilation is 10 cfm / person
+						per_person_ventilation_rate = OpenStudio.convert(10, 'ft^3/min', 'm^3/s').get
+
+						# assign remaining oa to per-area
+						new_oa_for_people_per_m2 = people_per_m2 * per_person_ventilation_rate
+						new_oa_for_people_cfm_per_f2 = OpenStudio.convert(OpenStudio.convert(new_oa_for_people_per_m2, 'm^3/s', 'cfm').get, '1/m^2', '1/ft^2').get
+						new_oa_for_people_cfm = number_of_people * new_oa_for_people_cfm_per_f2
+						remaining_oa_per_m2 = tot_oa_per_m2 - new_oa_for_people_per_m2
+						if remaining_oa_per_m2 <= 0
+						  runner.registerInfo("Space '#{space.name}' has #{number_of_people.round(1)} people which corresponds to a ventilation minimum requirement of #{new_oa_for_people_cfm.round(0)} cfm at 10 cfm / person, but total zone outdoor air is only #{tot_oa_cfm.round(0)} cfm. Setting all outdoor air as per-person.")
+						  per_person_ventilation_rate = tot_oa_per_m2 / people_per_m2
+						  dsn_oa.setOutdoorAirFlowperFloorArea(0.0)
+						else
+						  oa_per_area_per_m2 = remaining_oa_per_m2
+						  dsn_oa.setOutdoorAirFlowperFloorArea(oa_per_area_per_m2)
+						end
+						dsn_oa.setOutdoorAirFlowperPerson(per_person_ventilation_rate)
+						
+					end 
+			         end 
+		        end 
+				if add_econo
```

Comment:
Coordinate with @JanghyunJK on getting the EMS functionality from his measure into here. This will prevent the overcooling issues we saw with traditional E+ EMS controls.

Reply:
Yes, done.